samples/tfm_integration/tfm_ipc/boards/nrf9160dk_nrf9160ns.overlay

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* Modify the SRAM partitioning to accommodate the requirements
+ * for the Secure (TF-M) firmware for the configuration that is
+ * used in this sample.
+ */
+
+/* Increase the size of the Secure Firmware (TF-M).
+ * This modification is not required at the moment,
+ * since TF-M region definitions are configured
+ * statically in the TF-M project.
+ */
+&sram0_s {
+	reg = <0x20000000 DT_SIZE_K(192)>;
```

Comment:
Agreed, although this does not really play any role, at the moment, right? Since tFM has it's own build process that does not take DTS into account....

Reply:
done